backend/local/backend_plan_test.go

```diff
@@ -289,12 +289,9 @@ Terraform will perform the following actions:
 
   # test_instance.foo is tainted, so must be replaced
 -/+ resource "test_instance" "foo" {
-        ami = "bar"
+        # (1 unchanged attribute hidden)
 
-        network_interface {
-            description  = "Main network interface"
-            device_index = 0
-        }
+        # (1 unchanged block hidden)
     }
```

Comment:
This is a questionable looking output which is unfortunately at the top of the diff. My understanding is that the tests in this file are not representative of real diffs—in particular, the missing `id` argument is unrealistic. I decided to make minimal changes to allow tests to pass, rather than add `id` arguments to all of the resources throughout this test file.

Overall I think it is probably the right call to hide all unchanged non-identifying attributes in a tainted resource diff, but I'd be happy to hear suggestions for how to improve this.

Reply:
I think this looks fine - it'd probably be a good idea to add `id`s _in general_ so those tests are more realistic, but that's not something you need to do now. 

What's throwing me off is that I don't understand what's causing this specific test to opt-in to the experiment; from the files here I would expect the output to be unchanged. I'm probably staring right at the explanation, can you nudge me in the right direction?

Reply:
At present, the experiment is _opt-out_, so all tests are using the new concise renderer by default. My understanding is that opt-out is the goal for 0.14, unless we have insufficient or mixed feedback, in which case we may move to opt-in to work out those problems.

Reply:
Sorry, it's right in your original comment:
> The experiment is currently enabled by default

That's not what I expected, my mistake for skimming the comment! I had assumed we'd start with an opt-in model to get feedback and only after that enable it by default.